Backend/DataStructures.py

```diff
@@ -0,0 +1,64 @@
+
+# TODO: Add data classes and data structures to this file
+# Ex: Tweets class. User class
+
+
+# [User class]
+class User:
+
+    def __init__(self, uid, uname, uemail):
+        self.userId = uid
+        self.userName = uname
+        self.userEmail = uemail
+
+    def get_id(self):
+        return self.userId
+
+    def set_id(self, uid):
+        self.userId = uid
+
+    def get_username(self):
+        return self.userName
+
+    def set_username(self, uname):
+        self.userName = uname
+
+    def get_email(self):
+        return self.userEmail
+
+    def set_email(self, uemail):
+        self.userEmail = uemail
+# end User class
+
+
+class Tweet:
```

Comment:
there were some changes to the database design in Design Document II. This should probably add a few fields to reflect that

Reply:
Then again, if you don't think that's necessary at this point, please tell me and I'll just resolve this discussion